tests/ui-testing/pages/reportsPage.js

```
@@ -20,48 +20,81 @@
-    await this.page.locator(this.dateTimeButton).click();
-    await this.page.locator(this.absoluteTab).click();
-    await this.page.waitForTimeout(1000);
+  async setTimeToPast30Seconds() {
+    // Set the time filter to the last 30 seconds
+   // await expect(this.page.locator(this.dateTimeButton)).toBeVisible();
+    await this.page.waitForSelector(dateTimeButtonLocator);
+    await this.page.locator(this.dateTimeButton).click({ force: true });
+    await this.page.waitForSelector(relative30SecondsButtonLocator);
+
+    await this.relative30SecondsButton.click({ force: true });
+  }
```

Biome

[error] Duplicate class member name "setTimeToPast30Seconds"



(lint/suspicious/noDuplicateClassMembers)